docs/troubleshooting/windows-pv-tools.md

```diff
@@ -80,6 +69,76 @@ Consult your motherboard manual for details; for example, on Dell systems with i
 ![](../../static/img/performance-setting.png)
 </div>
 
+## Completely removing existing Xen PV drivers with XenClean
+
+XenClean is an utility for cleanly removing Xen PV drivers and management agents of the following products:
+
+* XCP-ng Windows PV Tools, versions 8.2 and 9.1
+* XenServer VM Tools for Windows, versions 9.3 and 9.4
+* Other Xen drivers
+
+It is included in the installation package of XCP-ng Windows PV Tools 9.1 and above.
+[See the newest releases here.](https://github.com/xcp-ng/win-pv-drivers/releases)
+
+To use XenClean, simply run the `Invoke-XenClean.ps1` script **as Administrator**. Your system will automatically reboot.
+
+**Note**: You should always use the latest version of XenClean regardless of your Windows PV driver version. The same applies to XenBootFix.
+
+**Note**: You should disable the "Manage Citrix PV drivers via Windows Update" option on your VM before running XenClean. Otherwise, Windows may reinstall PV drivers from Windows Update after rebooting.
+
+**Note**: If you downloaded XenClean from the internet, you may need to unblock the script file before running it. This can be done by right-clicking the file, then choosing **Properties** - **Unblock** - **OK**.
+
+**Tip**: XenClean leaves its log files at `%TEMP%\xenclean-<time>`. Please provide these logs in case of uninstallation failure.
```

Comment:
Use a callout?

```suggestion
:::tip
 XenClean leaves its log files at `%TEMP%\xenclean-<time>`. Please provide these logs in case of uninstallation failure.
:::
```

(better check the syntax, I'm writing it from memory)